configargparse.py

```diff
@@ -946,10 +947,14 @@ def format_help(self):
             msg += (" If an arg is specified in more than one place, then "
                 "commandline values override %s.") % (
                 " which override ".join(value_sources))
-        if msg:
-            self.description = (self.description or "") + " " + msg
 
-        return argparse.ArgumentParser.format_help(self)
+        text_width = max(self._get_formatter()._width, 11)
+        return textwrap.fill(msg, text_width)
+
+    def format_help(self):
+        msg = self._config_file_help()
+        return (argparse.ArgumentParser.format_help(self)
```

Comment:
don't need outer-most parentheses

Reply:
We do because the expression is split over two lines...